.circleci/config.yml

```diff
@@ -61,9 +64,24 @@ commands:
           path: test_root/log
       - store_test_results:
           path: reports
+      - run:
+          name: Coverage Report
+          command: |
+            pr_no=$(echo $CIRCLE_PULL_REQUEST | rev | cut -d/ -f1 | rev)
```

Comment:
Why do we need to reverse twice? Let's just split it and use the last item.

Reply:
I am not sure of this one as this is not my change, This is from existing code.